test/e2e/scheduling/predicates.go

```diff
@@ -126,7 +127,9 @@ var _ = SIGDescribe("SchedulerPredicates [Serial]", func() {
 		nodeToAllocatableMap := make(map[string]int64)
 		for _, node := range nodeList.Items {
 			allocatable, found := node.Status.Allocatable[v1.ResourceEphemeralStorage]
-			framework.ExpectEqual(found, true)
+			if !found {
+				framework.Failf("node ", node.Name, " not found to allocate storage")
```

Comment:
`Failf` with format string.

The message is misleading. The node was found, it just didn't have information about allocatable storage. Printing `node.Status` would be useful in this case.

Reply:
How about this?
`framework.Failf("unable to allocate to storage, node.Status: %v", node.Status)`

Reply:
This isn't about allocating storage.

The `node.Status.Allocatable[v1.ResourceEphemeralStorage]` map lookup checks whether such an entry exists in the map. I would just say `"node.Status.Allocatable %v does not contain entry %v", node.Status.Allocatable, v1.ResourceEphemeralStorage`